### DISCUSSION TITLE
Developer Documentation

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#178] [Ideas]
We need some clear and up to date developer documentation.  I created an issue for this.

We can have the discussion here.

@manuel12 @pawel975 @dalekvim @goutham03062001 

Where do we go with this?

**Comment by gbowne1** [comment]
I would like to propose we reduce developer and reviewer workload by limiting changes to only 250 lines of code instead of having a PR with 59 commits to it.  And to keep any changes to only within scope of an issue.  If there is no issue for a scope, we need to add that issue and make it clear that we stay in scope.

A second item, I will allow non breaking changes by direct if they are clear that they will not break and have passed CI or testing and include screen shots.

We still need some tooling enhancements, actions and workflows.

**Reply by dalekvim** [reply]
That’s cool. I can run tests with `npm test`, right? I don’t have experience writing tests though.

**Reply by gbowne1** [reply]
yes.  I don't have much experience with tests either @dalekvim.

**Reply by gbowne1** [reply]
@manuel12 Wrote the tests that we have which do login testing.
